test_files/enums.d

```diff
@@ -37,10 +37,11 @@ struct C
     _Anonymous_0 point;
 }
 
-enum _Anonymous_1
+alias i = C._Anonymous_0.i;
+alias j = C._Anonymous_0.j;
```

Comment:
Should this really be aliases at the module level? Aren't the enum members scoped in the struct?